src/api/consumer/lib/doctor.js

```diff
@@ -128,7 +128,7 @@ async function _generateExamineResultsTarFile(examineResults: ExamineResult[]):
     pack.entry({ name: '.bitmap' }, bitmap);
   }
   if (consumerInfo && consumerInfo.consumerConfig) {
-    pack.entry({ name: 'config.json' }, consumerInfo.consumerConfig.toJson());
+    pack.entry({ name: 'config.json' }, JSON.stringify(consumerInfo.consumerConfig.toPlainObject(), null, 4));
```

Comment:
Why did you use here 4 and not 2 from the constants?

Reply:
coz it's for the doctor, to be compliant with other json files there, which are 4. let me know if you want to change.

Reply:
anyway we should put it in constants